src/shared/domain/definition/case-field.model.ts

```diff
@@ -7,6 +7,7 @@ import { AccessControlList } from './access-control-list.model';
 // @dynamic
 export class CaseField implements Orderable {
   id: string;
+  hidden?: boolean;
```

Comment:
I would strongly suggest to rename this to `initiallyHidden`. The reason being that the hidden/show status on our UI is controlled by the show conditions not by this field. So a field can have `hidden` false but the is visible on the UI if the show condition triggers it